tests/test_plot.py

```diff
@@ -0,0 +1,78 @@
+"""
+Module used to test the plot functions
+"""
+
+import matplotlib
+import pandas as pd
+import xgboost as xgb
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+from hipe4ml import analysis_utils as au
+from hipe4ml import plot_utils as pu
```

Comment:
I would not use the the aliases since they may reduce readability, just for the tests.